docs/how/search-over-new-field.md

```diff
@@ -284,8 +298,43 @@ Response:
     "start": 0,
     "total": 1,
     "links": [
-      
+
     ]
   }
 }
 ```
+
+# Appendix: MidTier and UI changes
```

Comment:
One concern here is the context shift from the story shared in the backend section. We specifically took an example of adding a new search field `courses` in `CorpUserInfo` but then we are moving to dataset in the mid tier section. Do you think we can continue with the above example in this section?

Reply:
Sounds good